doorway-ui-components/src/page_components/help-center/DoorwayLinkableCardGroup.tsx

```diff
@@ -12,6 +12,9 @@ type DoorwayLinkableCardGroupProps = {
 const DoorwayLinkableCardGroup = (props: DoorwayLinkableCardGroupProps) => {
   const getLinks = () => {
     const links = []
+    if (!props.cards?.length) {
```

Comment:
This is just for extra safety in case the issue would happen again

Reply:
Wouldn't this happen naturally? Links is by default set to [], and in the for below, we just wouldn't hit it

Reply:
`props.cards` was coming back as undefined. 

And if you are interested the compiled code for `for(const x of y)` becomes `for (var _i = 0, _b = y; _i < _b.length; _i++)` So it's the `.length` that throws the null pointer

Reply:
Interesting! I feel like it just shouldn't enter it but that's good to know :)